From ee0dce4926b95c3c96217c076550216eb6dcd90b Mon Sep 17 00:00:00 2001
From: Steve French <stfrench@microsoft.com>
Date: Sat, 12 Dec 2020 12:49:28 -0600
Subject: [PATCH 157/197] cifs: remove some minor warnings pointed out by
 kernel test robot

Correct some trivial warnings caused when new file unc.c
was created. For example:

   In file included from fs/cifs/unc.c:11:
>> fs/cifs/cifsproto.h:44:28: warning: 'struct TCP_Server_Info' declared inside parameter list will not be visible outside of this definition or declaration
      44 | extern int smb_send(struct TCP_Server_Info *, struct smb_hdr *,

Reported-by: kernel test robot <lkp@intel.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
---
 fs/cifs/unc.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/fs/cifs/unc.c b/fs/cifs/unc.c
index 2c5665f5543a..394aa00cea40 100644
--- a/fs/cifs/unc.c
+++ b/fs/cifs/unc.c
@@ -7,7 +7,11 @@
  *              Jeff Layton <jlayton@kernel.org>
  */
 
+#include <linux/fs.h>
 #include <linux/slab.h>
+#include <linux/inet.h>
+#include <linux/ctype.h>
+#include "cifsglob.h"
 #include "cifsproto.h"
 
 /* extract the host portion of the UNC string */
-- 
2.27.0

